scripts/obtain/dotnet-install.ps1

```diff
@@ -50,7 +50,7 @@
 #>
 [cmdletbinding()]
 param(
-   [string]$Channel="preview",
+   [string]$Channel="rel-1.0.0",
```

Comment:
Yes, this is kind of the problem with 'dotnet-install' today, it is tied to the branch we are developing in.

The good news is that this "Publish Refactor" work that I'm doing is setting us up for success long term. After this, our publish output will be in the layout that we can maintain going forward.  So then the 'dotnet-install' script won't have to change anymore, once everything is in this new layout.  (I'm looking at you, `dotnet/core-setup`)